docs/user/management.asciidoc

```diff
@@ -58,12 +58,12 @@ years of historical data in combination with your raw data.
 | {ref}/transforms.html[Transforms]
 |Use transforms to pivot existing {es} indices into summarized or entity-centric indices.
 
-| <<managing-cross-cluster-replication, Cross-Cluster Replication>>
+| {ref}/ccr-getting-started.html[Cross-Cluster Replication]
 |Replicate indices on a remote cluster and copy them to a follower index on a local cluster.
 This is important for
 disaster recovery. It also keeps data local for faster queries.
 
-| <<working-remote-clusters, Remote Clusters>>
+| {ref}/ccr-getting-started.html[Remote Clusters]
```

Comment:
```suggestion
| {ref}/modules-remote-clusters.html[Remote Clusters]
```

Reply:
I think that this link should go to the [Remote clusters](https://www.elastic.co/guide/en/elasticsearch/reference/master/modules-remote-clusters.html) page.

Reply:
@lockewritesdocs I worry about sending users to a page that doesn't mention the UI.  Can you add content about the Kibana UI to the Remote clusters page?

Reply:
@gchaps, understood. Perhaps link them directly to the [Connect to a remote cluster](https://www.elastic.co/guide/en/elasticsearch/reference/master/ccr-getting-started.html#ccr-getting-started-remote-cluster) task instead?